docs/release/release_0_1.rst

```diff
@@ -0,0 +1,160 @@
+Announcement: napari 0.1.0
+==========================
+
+We're happy to announce the release of napari v0.1.0!
+napari is a fast, interactive, multi-dimensional image viewer for Python.
+It's designed for browsing, annotating, and analyzing large multi-dimensional
+images. It's built on top of Qt (for the GUI), vispy (for performant GPU-base
+rendering), and the scientific Python stack (numpy, scipy).
+
+This is our first minor release, timed for the 2019 SciPy Conference in Austin.
+It marks our transition from pre-alpha to alpha, and establishes a reasonable
+API for adding images, shapes, and other basic layer types to an interactive
+viewer. It supports launching a viewer with python scripting or from Jupyter
+notebooks.
+
+For more information, examples, and documentation, please visit our website:
+https://github.com/napari/napari
+
+Other Pull Requests
```

Comment:
If you're not going to do some cherry picking here, I suggest changing this heading to just "Pull Requests"

Reply:
will change